### PR TITLE
Math magicians: components

### PR DESCRIPTION
**Pull request by LoviMadaan**
### In this branch

- I used React components.
- I used React props.
- I used React state.

### CODE REVIEW

File: src/styles/Main.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+display: grid;
+justify-content: center;
+align-content: center;
+width: 400px;
+margin: 40px auto;
+grid-template-columns: repeat(4, 100px);
+grid-template-rows: minmax(120px, auto) repeat(5, 100px);
+box-shadow: 2px 2px 10px #333;
+border-radius: 10px;
+`;
```
Comment (iambenkis) [inline src/styles/Main.js]: - [ ] Good job creating the initial component of your calculator, but as required, your design should match the given pattern as showed on the picture below 😄 

| Pattern Design | Your design | 
| :---: | :---: |
| Capture |  Capture1 | 

What do you think about fixing that ?

File: src/styles/Main.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+
+export const Screen = styled.div`
+grid-column: 1/-1;
+background-color: rgba(60,63,67,0.75);
+justify-content: space-around;
```
Comment (iambenkis) [inline src/styles/Main.js]: - [ ] keep doing the great work 👍

File: src/Components/calculator.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import {
+  Container, Prevoius, Screen, Current, Button,
+} from '../styles/Main';
+
+const Calculator = () => (
+  <Container>
+    <Screen>
```
Comment (iambenkis) [inline src/Components/calculator.js]: - [ ] Keep doing the great work 👍

File: src/App.js [hunk 4]
```diff
@@ -1,27 +1,10 @@
...
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
+    <div className="calC">
+      <h1>Calculator App</h1>
+      <Calculator />
```
Comment (iambenkis) [inline src/App.js]: - [ ] Keep doing the great work

**Review by iambenkis — CHANGES_REQUESTED** [review]
# STATUS: REQUIRED CHANGES :recycle::recycle: 

Hi @LoviMadaan  
Good job so far! I know you worked hard to implement this project, I really appreciate the effort you put in! But there are some issues that you still need to work on to go to the next project but you are almost there! 

almost-there

## To highlight :champagne: 
- All linters are passing :heavy_check_mark: 
- Good commit history :heavy_check_mark: 
- Follow GitHub flow :heavy_check_mark: 

## Required changes :recycle: :recycle: 

- [ ] Check the comments under the review.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

_Feel free to leave any questions or comments in the PR thread if something is not 100% clear. 
Please ping me @iambenkis  when you comment, so I can receive the notification or use Slack with the same name._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @LoviMadaan , 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- Good commit messages.  ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**